src/plugins/timelist/TimelistComponent.vue

```diff
@@ -194,6 +194,7 @@ export default {
       if (this.isEditing) {
         this.filterValue = configuration.filter;
         this.hideAll = false;
+        this.setSort();
```

Comment:
We should add a regression test for this

Reply:
Added a unit test for this.